aleph0/src/createActions.ts

```diff
@@ -25,20 +42,45 @@ export const createActions = (
         properties: {},
       },
     },
-    readSnippetFile: {
-      function: async (args: { filePath: string }) => {
-        return { fileContents: readFile(args.filePath) };
+    expandSnippet: {
+      function: async (args: { id: string }) => {
+        const { file, references } = files[args.id];
+
+        const directory = process.cwd();
+
+        const filePath = `${directory}/${file}`;
```

Comment:
Same, create a separate function to read a snippet by id?d

Reply:
The function is tiny. I'd way rather just see 10 lines of code inline than have to jump about to debug this function in the future.

Reply:
Ok let's see if this pattern repeats in the future.